.buildkite/pipelines/es_serverless/verify_es_serverless_image.yml

```diff
@@ -17,8 +17,15 @@ agents:
 steps:
   - label: "Annotate runtime parameters"
     command: |
-      buildkite-agent annotate --context es-serverless-image --style info "ES Serverless image: $ES_SERVERLESS_IMAGE"
       buildkite-agent annotate --context kibana-commit --style info "Kibana build hash: $BUILDKITE_BRANCH / $BUILDKITE_COMMIT"
+      cat << EOF | buildkite-agent annotate --context es-serverless-image --style info
+        ES Serverless image: \`$ES_SERVERLESS_IMAGE\`
+
+        To run this locally:
+        \`\`\`
+        node scripts/es serverless --tag $ES_SERVERLESS_IMAGE
```

Comment:
Needs to be updated, $ES_SERVERLESS_IMAGE can either be a tag or an image

Reply:
Pushed https://github.com/elastic/kibana/pull/182579/commits/7c504130ff05125e1047152babd4262d80da0ae7 + 74e59698b4f576cf57293742bc9a8e8c391561e3

Reply:
Needs to run through tf first to be tested..will have to be after merge.